libs/webb-ui-components/src/constants/index.ts

```diff
@@ -60,6 +60,8 @@ export const WEBB_DAPP_NEW_ISSUE_URL =
 export const WEBB_FAUCET_URL = 'https://faucet.webb.tools/';
 export const STATS_DOCUS_URL =
   'https://docs.webb.tools/docs/projects/stats-dapp/overview/';
+export const POLKADOT_EXPLORER_URL =
+  'https://polkadot.js.org/apps/?rpc=wss://tangle-standalone-archive.webb.tools#/explorer/query/';
```

Comment:
The current setup is satisfactory for the time being. However, we should aim for a dynamic configuration based on the currently selected RPC. Please refer to the following links for more information:

- [populateBlockExplorer](https://github.com/webb-tools/webb-dapp/blob/6bad94a740ec2dcd63b139e4a33e878610609f90/libs/dapp-config/src/chains/substrate/index.tsx#L5-L14) in the Polkadot UI.
- [How to calculate the URL for a given block hash in the Polkadot UI](https://github.com/webb-tools/webb-dapp/blob/edb92734e13134fd9ebe1a678aa4b5f6c5b8caf7/libs/api-provider-environment/src/transaction/useTransactionQueue.tsx#L44-L47).